stdlib/public/core/Arrays.swift.gyb

```diff
@@ -473,11 +478,15 @@ public struct ${Self}<Element>
   /// For an instance of `${Self}`, `startIndex` is always zero. If the array
   /// is empty, `startIndex` is equal to `endIndex`.
 %end
+  @_inlineable
   public var startIndex: Int {
 %if Self == 'ArraySlice':
     return _buffer.startIndex
 %else:
-    return 0
+    @_inlineable
```

Comment:
Oops, the @_inlineable should apply to the getter too. Can you revert this and I'll make sure it works?